src/header/header_def.rs

```diff
@@ -0,0 +1,446 @@
+use crate::errors::RTPError;
```

Comment:
similar to h264_def.rs and mod.rs, it is not necessary to split to two files.

Reply:
Not really, header_def containers header and its definition, mod.rs contains constants and any other struct. We don't want a single file with an humongous code